extras/rya.export/README.md

```diff
@@ -0,0 +1,250 @@
+
+![Figure 1: Child to Parent Merge](resources/images/child_to_parent_merge.png)
+
+## Overview
+
+At deployment time, a Rya child instance will be created and sent out where its users will create, modify, and delete data. This data from the child will then need to be synced with its parent after deployment. By default, the data only needs to be handled at the datastore level and so will not need to know the logical consistency of the higher level models it's storing (i.e. Rya logic). Some decisions that were made about the tool:
+
+  * When copying, the parent is copying its data to its child and can optionally filter its data to be after a specified timestamp.
+
+  * When merging, the data is going only ONE way: the child is re-integrating its new data with the parent Rya instance (the parent does NOT have to merge its new data into the child).
+
+  * For Accumulo datastores, since an update to a row value is just a delete followed by an add, row value updates aren't needed.
+
+  * For Accumulo datastores, special care must be taken when there's a difference between the parent's and child's column visibility in the same row. In this case, the two visibility values should be OR'ed together to produce the new visibility value to use.
+  
+  * For Accumulo to MongoDB merging, column visibility is ignored when updating.
+
+  * The tool should support merging data from after a specific time (start time). This will merge in a smaller subset of data modifications from the child to the parent.
+  
+  * The Rya Export Client can handle copying/merging from Accumulo to Accumulo, Accumulo to MongoDB, MongoDB to Accumulo, and MongoDB to MongoDB.
+
+## Rya Export Client Process
+
+The Rya Export Client can be ran through the command line. It can either be used to copy a child instance from a parent or it can be used to sync an already exported child instance by merging its data over to the parent instance. The user can configure a timestamp for the data to be included so only data coming after that time is replicated. The command line will make calls to an uber jar containing the tool and execute the job. This will also allow extra config parameters (such as the location of the parent and child instance) to be used when called.
+
+## Implementation Details
+
+The Rya Export Client can be run on any machine connected to both the parent instance machine and the child instance machine. It determines which machine is the parent and child by storing metadata that marks the parent machine after the initial copy.
+
+The Accumulo side export uses a TimestampFilter iterator setting to only include the data that fits the time range. This time criteria is added to the Scanner that fetch the Rya statements to copy. The desired statements are fetched from the parent and then added to the child table using a BatchWriter. The process job also checks a parent's authorization list so they can be copied to the child user.
```

Comment:
This seems very implementation specific.  If this isn't for devs, I dont' see the point to specify this.